packages/nx-plugin/package.json

```
@@ -37,7 +37,8 @@
     "@nx/devkit": ">=17.0.0",
     "ansis": "^3.3.0",
     "nx": ">=17.0.0",
-    "zod": "^4.0.5"
+    "zod": "^4.0.5",
+    "chalk": "5.3.0"
```

We already have `ansis` for terminal colors, please use it instead of introudcing a new dependency.

```suggestion
    "zod": "^4.0.5"
```